sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -63,6 +71,10 @@ object Klaviyo {
         Registry.register<StateSideEffects>(StateSideEffects())
 
         Registry.get<State>().apiKey = apiKey
+
+        while (preInitQueue.isNotEmpty()) {
+            preInitQueue.poll()?.let { it() }
```

Comment:
TODO
- The operations in the queue are unprotected, so they should be try-catched here. 
- Log a message about retrying here.